.github/workflows/docker.yml

```diff
@@ -23,12 +29,91 @@ jobs:
     - name: Check out code into the Go module directory
       uses: actions/checkout@v2
 
+    - name: Log in to the GH Container registry
+      uses: docker/login-action@f054a8b539a109f9f41c372932f1ae047eff08c9
```

Comment:
nit/feel free to ignore: I see youre referencing multiple actions in this file with a sha. Its easier for future reviewers to see a version so they can quickly compare with the current action version.

Reply:
good point! I'll try to lock to a released version instead here.